### PR TITLE
Standalone build update

### PR DESCRIPTION
**Pull request by kurapov-peter**
Note: `make all_tests` didn't finish for me.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,6 +86,19 @@ The following `cmake`/`ccmake` options can enable/disable different features:
                               **Note:** this is a >50MB download.
 - `-DPREFER_STATIC_LIBS=off` - Static link dependencies, if available. Default is `off`.
 
+# Engine standalone build
```
Comment (ienkovich) [inline README.md:89]: This build option is not specifically for the embedded engine, you get the regular server as well. You also don't need to go this way to get the embedded engine and can simply using -DENABLE_DBE=ON. So, I would call this section 'build in conda environment` or similar.

File: README.md [hunk 2]
```diff
@@ -86,6 +86,19 @@ The following `cmake`/`ccmake` options can enable/disable different features:
...
+
+Engine standalone (resides in "modin" branch) build requires conda installation:
```
Comment (ienkovich) [inline README.md:91]: We shouldn't mention our working branches in the documentation. All related changes are pushed into master branch sooner or later. Also, none of our features require conda. We just use conda for easier dependencies installation.

File: README.md [hunk 3]
```diff
@@ -86,6 +86,19 @@ The following `cmake`/`ccmake` options can enable/disable different features:
...
+
+`git checkout modin`
```
Comment (ienkovich) [inline README.md:93]: This shouldn't be in build instruction.

File: README.md [hunk 4]
```diff
@@ -94,6 +107,12 @@ The `sanity_tests` target runs the most common tests. If using Makefiles to buil
 
     make sanity_tests
 
+# Engine standalone unit testing
```
Comment (ienkovich) [inline README.md:110]: Build in conda environment doesn't change testing procedure. The embedded engine also doesn't have any special procedures. So, this section is not needed.

**Review by ienkovich — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (kurapov-peter) [reply README.md]: I left the section as is, but joined with general one. Should I remove it completely? I haven't found any mention of how to run it in the readme.

**Review by ienkovich — APPROVED** [review]

**Pull request merged**